.github/workflows/package.yaml

```diff
@@ -11,13 +11,6 @@ on:
 jobs:
   build-and-push-image:
     runs-on: ubuntu-latest
-
```

Comment:
This was a leftover from using GitHub registry. It is not needed for the dockerhub push